src/apps/workspaces/crud/user_applet_access.py

```diff
@@ -618,16 +605,77 @@ async def get_workspace_managers(
             .correlate(AppletSchema)
         )
 
-        query: Query = (
+        invited_users = (
+            select(
+                InvitationSchema.id,
+                InvitationSchema.first_name,
+                InvitationSchema.last_name,
+                InvitationSchema.email,
+                func.array_agg(
+                    aggregate_order_by(
+                        func.distinct(InvitationSchema.title),
+                        InvitationSchema.title,
+                    )
+                ).label("titles"),
+                func.array_agg(
+                    aggregate_order_by(
+                        func.distinct(InvitationSchema.role),
+                        InvitationSchema.role,
+                    )
+                ).label("roles"),
+                func.array_agg(
+                    aggregate_order_by(
+                        func.json_build_object(
+                            text("'applet_id'"),
+                            AppletSchema.id,
+                            text("'applet_display_name'"),
+                            AppletSchema.display_name,  # noqa: E501
+                            text("'applet_image'"),
+                            AppletSchema.image,
+                            text("'role'"),
+                            InvitationSchema.role,
+                            text("'encryption'"),
+                            AppletSchema.encryption,
+                        ),
+                        AppletSchema.id,
+                    )
+                ).label("applets"),
+                InvitationSchema.status,
+                InvitationSchema.created_at,
+                InvitationSchema.user_id,
+            )
+            .select_from(InvitationSchema)
+            .join(
+                AppletSchema,
+                and_(
+                    AppletSchema.id == InvitationSchema.applet_id,
+                    AppletSchema.soft_exists(),
+                ),
+            )
+            .where(
+                InvitationSchema.role != Role.RESPONDENT,
+                InvitationSchema.status == InvitationStatus.PENDING,
```

Comment:
we should review it with designers

Reply:
owners doesn't have an invitation, so no invitation date to show, we should review it and show dates only for pending

Reply:
we could handle this in properly ticket

Reply:
This is why I suggested separating the `created_at` and `invited_at` fields so that on the FE we can use `created_at` for users who don't have invitations like owners.

I understand that showing a date called "Invitation Date" for users who technically weren't invited might be misleading, so I'll run that by the designers for feedback.

Still, for those users with an approved invitation, we'll need to return the data for them so we can display it on the FE

Reply:
I wanna keep this comment thread focused on including accepted invitations, so I'll respond [here](https://github.com/ChildMindInstitute/mindlogger-backend-refactor/pull/1403#discussion_r1637927064)

Reply:
Ok I confirmed with Matt that we should be including approved invitations so we can show the original invitation date on the FE. That means we'll also need to split `created_at` and `invited_at`

Reply:
no, I will not include it

Reply:
we should keep abstraction what is an user and what is abstraction, this what is this ticket, not two lists.

Reply:
show created_at will do what you need in FE, BE  will decide with date